bin/ghe-backup

```diff
@@ -37,6 +37,25 @@ while true; do
   esac
 done
 
+# Logging functions
+logCommon(){
+	local logLevel=$1
+	local message=$2
+	echo "`date +\"%Y-%m-%d %T\"` - $logLevel - $message"
+}
+
+logInfo(){
+	logCommon "INFO" "$1"
+}
+
+logWarn(){
+	logCommon "WARN" "$1"
+}
+
+logError(){
+	logCommon "ERROR" "$1"
+}
+
```

Comment:
These log functions should be moved to the `share/github-backup-utils/ghe-backup-config` common file and the `ghe-restore` script updated to use them so that we have consistent logging.

Reply:
Moved log functions to `share/github-backup-utils/ghe-backup-config`